.github/workflows/ci.yml

```diff
@@ -158,7 +159,8 @@ jobs:
             --disable-valgrind \
             --with-oniguruma=builtin \
             --enable-static \
-            --enable-all-static
+            --enable-all-static \
+            CFLAGS="-O2 -pthread -fstack-protector-all"
```

Comment:
Doh i somehow assumed `-fsanitize=safe-stack` was clang's versions of `-fstack-protector-all` but seems to be two different stack protection techniques.

I still haven't managed to get `-fsanitize=safe-stack` to work for darwin x86, but do seems to work on linux.